packages/astro/src/core/build/static-build.ts

```diff
@@ -57,46 +57,54 @@ export async function staticBuild(opts: StaticBuildOptions) {
 			const [renderers, mod] = pageData.preload;
 			const metadata = mod.$$metadata;
 
-			// Track client:only usage so we can map their CSS back to the Page they are used in.
-			const clientOnlys = Array.from(metadata.clientOnlyComponentPaths());
```

Comment:
There's going to be merge conflicts after https://github.com/withastro/astro/pull/3625/ is merged. However all of this code goes away, so I think your changes won't be needed.

Reply:
Yep, noticed this! Was glad to see this going away.